lib/collection/src/operations/types.rs

```diff
@@ -338,8 +338,10 @@ pub struct CollectionClusterInfo {
 pub struct ShardTransferInfo {
     pub shard_id: ShardId,
 
+    /// Target shard ID if different than source shard ID
+    ///
+    /// Used exclusively with `ReshardStreamRecords` transfer method.
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    #[schemars(skip)] // TODO(resharding): expose once we release resharding
     pub to_shard_id: Option<ShardId>,
```

Comment:
This one is fine and useful, though. (Cause support would probably need this, when resharding explodes and people come to complain.) 👍